README.md

```diff
@@ -1 +1,70 @@
+# Quix CLI
+
+A way of accessing quix features via CLI
```

Comment:
can we use chat GPT to improve the Readme. Here an example that I have just did:

![image](https://github.com/quixio/quix-cli/assets/79307293/618383d0-bb41-420d-8fc7-7773153a4195)

![image](https://github.com/quixio/quix-cli/assets/79307293/66f942c8-23e1-4caf-bb1b-2bc5f17b6fa4)


![image](https://github.com/quixio/quix-cli/assets/79307293/aa791fe0-f465-4c46-ae2a-02f5be38934a)

![image](https://github.com/quixio/quix-cli/assets/79307293/7aea41f0-7d27-4a15-9a9c-ccf936f8987b)